addons/l10n_in_withholding/views/account_move_views.xml

```diff
@@ -33,11 +33,24 @@
                     </field>
                 </page>
             </xpath>
-            <xpath expr="//sheet" position="before">
-                <div class="alert alert-warning" role="alert" invisible="not l10n_in_tcs_tds_warning">
-                    <field name="l10n_in_tcs_tds_warning" readonly="1"/>
-                </div>
-            </xpath>
+        </field>
+    </record>
+
+    <record id="view_move_line_list_l10n_in_withholding" model="ir.ui.view">
+        <field name="name">account.move.line.list.l10n.in.withholding</field>
+        <field name="model">account.move.line</field>
+        <field name="arch" type="xml">
+            <list string="Journal items" editable="top" create="0" default_group_by="l10n_in_tds_tcs_section_id">
+                <field name="product_id" readonly="1"/>
+                <field name="name" widget="section_and_note_text"/>
+                <field name="account_id"/>
+                <field name="tax_ids"
+                       widget="many2many_tags"
+                       domain="[('type_tax_use', '=', context.get('default_tax_type_use'))]"
+                       column_invisible="context.get('move_type')"/>
```

Comment:
Why do not we show it for the in_invoice case?

Reply:
TDS is applied from wizard not in `account.move.line`